doc/sphinx/generate_python_api_doc.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# 
+# This script generates stub files for automatically generated documentation.
+# 
+###############################################################################
+
+# paths relative to this file location
+NGRAPH_REPO=../..
+DOC_DIR=${NGRAPH_REPO}/doc
+TMP_DIR=/tmp/sphinx_auto_py_doc
+
+
+cd ${NGRAPH_REPO}/python/ngraph
+PYTHONPATH=. sphinx-autogen -t ${DOC_DIR}/sphinx/source/_templates/ -o ${TMP_DIR} ${DOC_DIR}/sphinx/source/python_api/structure.rst
+
+EXCLUDE_DIRS="${NGRAPH_REPO}/python/ngraph/impl* 
+              ${NGRAPH_REPO}/python/ngraph/utils*"
+CURRENT_DIR=.
```

Comment:
Can we keep all the lines setting vars together at the top?

Reply:
Yes.